.github/workflows/cd.yml

```
@@ -0,0 +1,206 @@
+            Автор: ${{ github.actor }}
+            Pull request: ${{ github.event.pull_request.html_url || 'N/A' }}
+      - name: Notify Discord про статус CI (failure)
+        if: failure()
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+        uses: Ilshidur/action-discord@0.3.2
+        with:
+          args: |
+            CI failed для ${{ github.repository }} на гілці ${{ github.ref }}.
+            Автор: ${{ github.actor }}
+            Pull request: ${{ github.event.pull_request.html_url || 'N/A' }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)